internal/pkg/cli/deploy/env.go

```diff
@@ -51,6 +54,37 @@ type envDeployer struct {
 	uploadCustomResourceFlag bool
 }
 
+type NewEnvDeployerInput struct {
+	App             *config.Application
+	Env             *config.Environment
+	SessionProvider *sessions.Provider
+}
+
+func NewEnvDeployer(in *NewEnvDeployerInput) (*envDeployer, error) {
```

Comment:
Can we add a comment to this public function? Also, where are we calling this function from?

Reply:
https://github.com/aws/copilot-cli/blob/0b94c7f4a57b4dd5724331352626c7908553fb0b/internal/pkg/cli/env_deploy.go#L88 Here 😄